tests_react_native/specs/notification.spec.ts

```diff
@@ -250,6 +250,42 @@ export function NotificationSpec(spec: TestScope): void {
           });
         },
       );
+
+      spec.it('displays a notification with a quick action without input', async function () {
+        return new Promise(async (resolve, reject) => {
+          return notifee
+            .displayNotification({
+              title: '',
+              body: '',
+              android: {
+                channelId: 'high',
+                actions: [
+                  {
+                    title: 'First Action',
+                    pressAction: {
+                      id: 'first_action',
+                    },
+                  },
+                ],
+              },
+            })
+            .then(id => {
+              expect(id).equals(id);
+              resolve();
+            })
+            .catch(e => {
+              reject(e);
+            });
+        });
+
+        // Manual steps:
+        // 1. Minimize app
+        // 2. Open notification drawer
+        // 3. Tap on 'First Action'
+        // 4. Make sure you see:
+        // >  WARN  Received a ACTION_PRESS Background event in JS mode.
+        // >  WARN  Notification Cancelled first_action
```

Comment:
Not sure how you guys test notification tapping. On Android it is possible to simulate some actions with `adb` but that's not robust since you need to know exact point for tapping, e.g.:
```sh
adb shell input keyevent KEYCODE_HOME
adb shell cmd statusbar expand-notifications
adb shell input tap 990 1269
```

Note that on Android 9 this test passes without the fix in `isAppInForeground`.
On Android 12, however, the fix is necessary for this test to pass.
Hence the fix is needed for this test to pass on both Android 7 and Android 12